Automattic/wp-calypso
# Pattern Library: Mobile tweaks

Fixes https://github.com/Automattic/dotcom-forge/issues/6278
Fixes https://github.com/Automattic/dotcom-forge/issues/6389

## Proposed Changes

This PR fixes a few small things, primarily on mobile:

- When searching, the `%(count) patterns` heading is now displayed on mobile
- The `Copy, paste, customize` and `All about patterns` sections now use scroll snapping because it's neat
- When searching in all categories, the `All categories` link remains active (this was primarily problematic on mobile, where the dropdown previously didn't have any selected text while searching all categories)


https://github.com/Automattic/wp-calypso/assets/1101677/fba3f4e6-0e46-4c32-97fd-9c87c4d69136



## Testing Instructions



1. Navigate to `/patterns` on mobile. I suggest using the Calypso preview link on your phone.
2. Search for something
3. Ensure that an `N patterns` heading is shown and that the nav dropdown still says `All categories`
4. Navigate to a category
5. Ensure that no `Patterns` heading is shown
6. Go back to `All categories`
7. Scroll down to the `Copy, paste, customize` section and ensure that it now has scroll snapping
